eng/templates/Azure.ServiceTemplate.Template/samples/README.md

```diff
@@ -0,0 +1,30 @@
+---
+page_type: sample
```

Comment:
This PR: https://github.com/Azure/azure-sdk-for-net/pull/26484
Has really nice content wrt samples .. let's use that instead of this.  Thanks!

Reply:
updated